src/components/datagrid/controls/data_grid_toolbar_control.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import { EuiButtonEmpty } from '../../button';
+import { EuiFlexGroup, EuiFlexItem } from '../../flex';
+import { EuiNotificationBadge } from '../../badge';
+import { useEuiI18n } from '../../i18n';
+import { EuiDataGridToolbarControlProps } from '../data_grid_types';
+
+export const EuiDataGridToolbarControl = ({
+  children,
+  badgeContent,
+  ...buttonProps
+}: EuiDataGridToolbarControlProps) => {
+  const badgeAriaLabel = useEuiI18n(
+    'euiDataGridToolbarControl.badgeAriaLabel',
+    'Current count: {count}',
```

Comment:
Minor copy suggestion - based on what I'm seeing in the screenshots, I think maybe a more accurate description might be "Active" instead?
```suggestion
    'Active: {count}',
```

Screen readers also read this out as, e.g. "six slash eight" instead of "six out of eight" which I would personally prefer for ease of understanding. I might try to finagle that a bit here if it doesn't take too much time 🤔

Reply:
Screen reader text improved in https://github.com/elastic/eui/pull/7369/commits/f4d0c88d22c480ac62543facea720e72994d3e28